src/tests/ftest/vmd/led.py

```diff
@@ -0,0 +1,161 @@
+"""
+  (C) Copyright 2020-2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import json
+import time
+from osa_utils import OSAUtils
+from avocado.utils import process
```

Comment:
As a result of handling a CommandFailure  exception this is no longer needed:
```suggestion
```

Reply:
Done.